pkg/daemons/control/server.go

```diff
@@ -200,6 +204,10 @@ func apiServer(ctx context.Context, cfg *config.Control) error {
 	argsMap["enable-admission-plugins"] = "NodeRestriction"
 	argsMap["anonymous-auth"] = "false"
 	argsMap["profiling"] = "false"
+	if cfg.MultiClusterCIDR {
+		argsMap["feature-gates"] = argsMap["feature-gates"] + ",MultiCIDRRangeAllocator=true"
```

Comment:
use this: https://github.com/k3s-io/k3s/blob/7739c8b97ee453a82d2762f2c2172c64e4aaa825/pkg/util/gates.go#L3-L4